packages/auth/src/core/util/handler.ts

```diff
@@ -129,8 +129,11 @@ export async function _getRedirectUrl(
 
 function getHandlerBase({ config }: AuthInternal): string {
   if (!config.emulator) {
-    return `https://${config.authDomain}/${WIDGET_PATH}`;
```

Comment:
This will now hardcode http:// for the localhost authDomain. It is possible that this doesn't work for some folks who are setting up TLS on localhost. But i think it is worth merging this and seeing what the feedback is.

Also adding @renkelvin to take a look.

Reply:
Good point. I think the fix Parijat just made should fix that.